v7.4.2 breaks cherry-picking import of DayPicker

**Describe the bug**
A clear and concise description of what the bug is.
since the release of v7.4.2, we started receiving the following error in our projects:
`Module not found: Error: Can't resolve 'react-day-picker/DayPicker'`

**To Reproduce**
simply use the following import statements:
`import DayPicker from 'react-day-picker/DayPicker';`

**Expected behavior**
This used to work up until v7.4.1, so I'd expect this to work and break only in the next major version.

**Additional context**
probably caused by this commit: https://github.com/gpbl/react-day-picker/commit/e1afc17b6b556e9e57f6bc14d87662fb6c875d07

--- a/DayPicker/index.js
+++ b/DayPicker/index.js
@@ -1,0 +1,1 @@
+module.exports = require('../build/DayPicker');

--- a/DayPicker/package.json
+++ b/DayPicker/package.json
@@ -1,0 +1,3 @@
+{
+  "types": "../types/DayPicker.d.ts"
+}

--- a/package.json
+++ b/package.json
@@ -8,6 +8,7 @@
   "style": "lib/style.css",
   "files": [
     "build",
+    "DayPicker",
     "DayPickerInput",
     "lib",
     "moment",

